server/apps/main/templates/main/confirmation_page.html

```diff
@@ -0,0 +1,322 @@
+    div.btn_flex_two {
+      display: flex;
+      max-width: 500px;
+      margin: auto;
+      justify-content: space-between;
+    }
+
+    div.btn_flex_one button,
+    div.btn_flex_two button {
+      
+    }
+
+    div.btn_flex_one button a,
+    div.btn_flex_two button a {
+      color: #000;
+      text-decoration: none;
+    }
+
+    @media screen and (max-width: 750px){
+      div.btn_flex_one,
+      div.btn_flex_two {
+        flex-direction: column;
+      }
+
+      div.btn_flex_one button,
+      div.btn_flex_two button{
+        margin-bottom: 10px;
+      }
+    }
+
+  </style>
+  <!-- Custom Styles Ends Here -->
+
+
+  <!-- Navbar Starts Here -->
+  <nav class="navbar navbar-expand">
+    <ul class="navbar-nav mx-auto justify-content-around">
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="{% url "main:index" %}">About</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="{% url "main:upload" %}">Share</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="#">View</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="#">MyPage</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="{% url "main:logout" %}">Logout</a>
+      </li>
+    </ul>
+  </nav>
+  <!-- Navbar Ends Here -->
+
+
+  <!-- Container Starts Here -->
+  <div class="container">
+
+    <!-- Confirmation Section Starts Here -->
+    <section class="confirmation">
+      <div class="confirmation_flex">
+
+        <!-- Confirm SVG -->
+        <div class="confirm_svg">
+          <i class="fa fa-check" aria-hidden="true"></i>
+        </div>
+
+        <!-- Confirm Success -->
+        <div class="confirm_success">
+          <p class="h3 text-success">
+            Success!
+          </p>
+
+          <p class="h5">
+            Thank you for sharing your experience.
+          </p>
+
+          <!-- What Happens Next Dropdown Goes Here -->
+          <div class="dropdown show what_happens">
+            <a class="dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              What happens next?
+            </a>
+
+            <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+              <a class="dropdown-item" href="#">Action</a>
+              <a class="dropdown-item" href="#">Another action</a>
+              <a class="dropdown-item" href="#">Something else here</a>
+            </div>
+          </div>
+          <!-- What Happens Next Dropdown Ends Here -->
+        </div>
+      </div>
+
+
+      <!-- What Happens Now Section Starts Here -->
+      <div class="what_now">
+        <p class="h4" id="whatNow">
+          What Happens Now? <span class="step">(step by step)</span>
+        </p>
+
+        <div class="what_now_texts">
+          <ul class="texts card d-none" id="whatNowTexts">
```

Comment:
Very nice usage of `d-none`!! 🥳 

For the sake of visibility, can we remove this so that others can see the text items you've added below? 
In milestone 4, we can add CSS magic to hide/show as needed. :)